installer/dual_rootfs_device.go

```diff
@@ -28,6 +28,14 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+const (
+	verifyRebootError = "Reboot to new update failed. " +
```

Comment:
reword to "_Reboot to the new update failed_"

Reply:
Will change!